### PR TITLE
Update ts3admin.class

### PR DESCRIPTION
**Pull request by Pantoflarz**
Changed selectServer, to enable "use client_nickname=" as per server version 3.4.0 -
https://forum.teamspeak.com/threads/137292-Release-TeamSpeak-3-Server-3-4-0

### CODE REVIEW

File: lib/ts3admin.class.php [hunk 1]
```diff
@@ -3263,11 +3263,12 @@ private function loadQueryData()
   * @param		boolean	$virtual	set true to add -virtual param [optional]
   * @return     array success
   */
-	function selectServer($value, $type = 'port', $virtual = false) { 
+	function selectServer($value, $type = 'port', $virtual = false, $name = null) { 
+		if($name == null && strlen($name) >=3 ){ $name = ''; }else{ $name = " client_nickname=".$name.""; }
```
Comment (MINORITYmaN) [inline lib/ts3admin.class.php]: if `$name` equals `null` then what result would be `strlen($name)`  :face_with_head_bandage: 
rethink this line

Comment (MINORITYmaN) [reply lib/ts3admin.class.php]: take a look at https://github.com/par0noid/ts3admin.class/blob/master/lib/ts3admin.class.php#L4253
and rethink

Comment (lgund) [reply lib/ts3admin.class.php]: You don't need to check if the value is Null cause a strlen of Null is 0.

**Comment by par0noid** [comment]
Thanks!

**Pull request merged**